modeling/model_configuration/classes.attributes.json

```diff
@@ -149,6 +160,35 @@
     "id": "3T-SBOM-EMS-Artifact-Document-signature",
     "reverseName": "document"
   },
+  {
+    "class": "ExternalDocumentRef",
+    "name": "identifier",
+    "description": "The identifier of the document, unique within the namespace.",
+    "type": "http://www.omg.org/spec/UML/20131001/PrimitiveTypes.xmi#String",
+    "multiplicity": "1",
+    "package": "Artifact",
+    "id": "3T-SBOM-EMS-Artifact-ExternalDocumentRef-identifier"
+  },
+  {
+    "class": "ExternalDocumentRef",
+    "name": "namespace",

```

Comment:
Just a note that this term is incompatible with SPDX which is spdxDocument.  If this proposal is accepted, we should add an issue to SPDX 3.0 to change spdxDocument -> namespace.